test/http-test.js

```diff
@@ -416,6 +416,26 @@ describe("httptest", () => {
 
       expect(resp.body).to.equal(JSON.stringify({ a: "b" }));
     });
+
+    it("content-type application/json returns body as json", async () => {
+      const resp = await request(app)
+        .post("/api")
+        .send({ a: "b" })
+        .expect("content-type", "application/json; charset=utf-8")
+        .expect(201);
+
+      expect(resp.body).to.deep.equal({ a: "b" });
+    });
+
+    it("content-type application/json returns body as JSON", async () => {
+      const resp = await request(app)
+        .post("/api")
+        .send({ a: "b" })
+        .expect("content-type", "application/json; charset=utf-8")
+        .expect(201);
+
+      expect(resp.body).to.deep.equal({ a: "b" });
+    });
```

Comment:
can't see the difference between this test and the one above other than json being uppercased in this description. am I missing something?